src/unix/sock_ctrl_msg.rs

```diff
@@ -98,7 +104,13 @@ impl CmsgBuffer {
 
 fn raw_sendmsg<D: IntoIovec>(fd: RawFd, out_data: &[D], out_fds: &[RawFd]) -> Result<usize> {
     // SAFETY: CMSG_SPACE is a pure calculation. We ensure that the input value does not exceed the range of c_uint
-    let cmsg_capacity = unsafe { CMSG_SPACE(size_of_val(out_fds).try_into().unwrap()) as usize };
+    let cmsg_capacity = unsafe {
+        CMSG_SPACE(
+            size_of_val(out_fds)
+                .try_into()
+                .expect("size_of_val(out_fds) exceeds u32 range"),
+        ) as usize
+    };
```

Comment:
This change belongs to commit 2. As I mentioned, instead of using `.unwrap()` use `.expect()`.